com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/clients/EventsGetter.java

```diff
@@ -0,0 +1,56 @@
+package com.sap.cloud.lm.sl.cf.core.cf.clients;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import javax.inject.Inject;
+
+import org.cloudfoundry.client.lib.CloudControllerClient;
+import org.cloudfoundry.client.lib.domain.CloudEvent;
+import org.cloudfoundry.client.lib.util.CloudEntityResourceMapper;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class EventsGetter extends CustomControllerClient {
+
+	private static final String GUID = "guid";
+	private static String EVENTS_URL = "/v2/events";
+	private CloudEntityResourceMapper resourceMapper = new CloudEntityResourceMapper();
```

Comment:
The CloudEntityResourceMapper is being used a lot in the other implementations of the CustomControllerClient. Could we make the logic generic?